specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/preview/2024-03-01-preview/diagnostic.json

```
@@ -0,0 +1,544 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ComputeDiagnosticResourceProviderClient",
+    "description": "The Compute Diagnostic Resource Provider Client",
+    "version": "2021-06-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/diagnosticOperations/{operationId}": {
+      "get": {
+        "tags": [
+          "diagnosticOperations"
+        ],
+        "operationId": "DiagnosticOperations_Read",
+        "description": "Gets status of a Diagnostic operation issued for Disk Inspection.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationParameter"
+          },
+          {
+            "in": "path",
+            "name": "operationId",
+            "required": true,
+            "type": "string",
+            "description": "The identifier pointing to a request for Disk Inspection. This is used for polling result of the request."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```

same as other places
```suggestion
             "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
```

---

+1

---

Updated